android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -909,9 +908,13 @@ private void runFeed(boolean isNewContent, Feed feed) {
         }
 
         mFeedHash = feed.hash;
-        mNtpAdapter.notifyItemRangeRemoved(
-                mNtpAdapter.getStatsCount() + mNtpAdapter.getTopSitesCount() + 1,
-                mNewsItemsFeedCard.size());
+        int newsItemsFeedCardSize = mNewsItemsFeedCard.size();
+        new Handler(Looper.getMainLooper()).post(() -> {
```

Comment:
@sujitacharya2005 should we check for `if (mNewsItemsFeedCard != null && mNewsItemsFeedCard.size() > 0) ` here ?